src/types/rule.d.ts

```diff
@@ -217,3 +213,5 @@ export interface RuleMetrics {
 }
 
 export type RuleDataItemWithMetrics = Merge<RuleItem, { metrics: Metrics }>
+
+export type BridgeItem = HTTPBridge & MQTTBridge
```

Comment:
The `&` operator doesn't seem very accurate here, but it seems to reduce the number of errors reported?🤔

Reply:
I don't feel like I can say it's inaccurate, it's more likely to be a dynamic type here, but if it's a union type, it has to be asserted every time, but it's not fixed every time here either, it's based on the value of the API bounce back, so I didn't think of a way to solve the type for now and used `&`

Reply:
I change back.